zeek-transport-default.yaml

```diff
@@ -0,0 +1,72 @@
+Days: 7
+Logs:
+        #These headings are for user friendliness, they do not do anything
+        Default:
```

Comment:
Since the headings don't do anything, I think it would be less confusing to make all the individual log files nested directly under `Logs:` (so de-indent one level) and make the `Default:`, `Troubleshooting:`, and `All:` comments instead of YAML keys.